src/TextInput/TextInput.tsx

```diff
@@ -71,28 +57,27 @@ export const TextInput = forwardRef(function TextInput(
     onInputChange,
     disabled = false,
     required = false,
+    renderAsTitle = false,
   }: TextInputProps,
```

Comment:
We shouldn't have to declare props that are just passed to `Field` manually
I think we can just spread the rest
```suggestion
    disabled = false,
    ...fieldProps
  }: TextInputProps,
```

That way, with new `Field` API update, we wouldn't need to update any other components as everything should be handled by default

Reply:
I'm not sure I follow - with the existing pattern we don't need to update any existing components, as the  new prop is optional and false as default?

Reply:
Yes you can just spread the props that aren't actually needed inside the input components:
```ts
const Component = ({ error, ...fieldProps }) => {
  return (
    <Field {...fieldProps}>
      <Input  error={error} />
    </Field>
  )
}
```

In future updates of `Field`, if we add a new prop, it's going to be passed automatically to all component that consumes `Field` and the typing would be correct as well since the types are shared

Reply:
Fixed in [8b4f6ed](https://github.com/marshmallow-insurance/smores-react/pull/2023/commits/8b4f6ed40ca0c108946ed7259b04ad958ba7315a)